etc/config/cobol.defaults.properties

```
@@ -0,0 +1,15 @@
+# Default settings for COBOL
+compilers=&gnucobol
+defaultCompiler=gnucoboldefault
+compilerType=gnucobol
```

```suggestion
compilerType=cobol
```
this is needed to get the server to start up

---

Too bad we have a lang name used to denote a compiler... Only brings confusion later on, when we have several compilers for the same lang :'(

---

we can rename the compilerType instead